codelists/views/index.py

```diff
@@ -9,46 +10,57 @@
 
 
 def index(request, organisation_slug=None, status=Status.PUBLISHED):
-    handles = Handle.objects.filter(is_current=True)
+    tracer = trace.get_tracer(__name__)
```

Comment:
We probably want this at the top since `__name__` is the module name, that matches how loggers are typically invoked in Python too.